Sources/Player/Player.swift

```diff
@@ -360,6 +360,26 @@ public extension Player {
 		}
 	}
 
+	func offline(mediaProduct: MediaProduct) -> Bool {
+		offlineEngine?.offline(mediaProduct: mediaProduct) ?? false
+	}
+
+	func deleteOffline(mediaProduct: MediaProduct) -> Bool {
+		offlineEngine?.deleteOffline(mediaProduct: mediaProduct) ?? false
+	}
+
+	func deleteAllOffline() -> Bool {
```

Comment:
[suggestion] I guess similar to above, just offline is not clear what it is offline, so maybe:
```suggestion
	func deleteAllOfflinedMediaProducts() -> Bool {
```
or
```suggestion
	func deleteAllOfflinedItems() -> Bool {
```
or something similar. I guess we make it consistent to how we name above.